android/src/main/java/com/datadog/reactnative/DdSdkConfigurationExt.kt

```diff
@@ -20,7 +20,8 @@ fun ReadableMap.asDdSdkConfiguration(): DdSdkConfiguration {
         sampleRate = getDouble("sampleRate"),
         site = getString("site"),
         trackingConsent = getString("trackingConsent"),
-        additionalConfig = getMap("additionalConfig")?.toHashMap()
+        additionalConfig = getMap("additionalConfig")?.toHashMap(),
+        manualTracingEnabled = getBoolean("manualTracingEnabled")
```

Comment:
Am I wrong or we forgot to bump the `bridge` version here ?

Reply:
this PR to be updated, I marked it as draft for now